lib/elixir/lib/inspect/algebra.ex

```diff
@@ -283,6 +311,23 @@ defmodule Inspect.Algebra do
     fold_doc(docs, &concat(&1, &2))
   end
 
+  @doc """
+  Colors a document if a the identifier has a color in the options.
+  """
+  @spec color(color_identifier, t, Inspect.Opts.t) :: doc_color
+  def color(identifier, doc, opts \\ %Inspect.Opts{}) when is_doc(doc) do
+    precolor = Inspect.Opts.syntax_color(opts, identifier)
+    if precolor do
+      postcolor = Inspect.Opts.syntax_color(opts, :default)
+      concat(
+        doc_color(doc, precolor),
```

Comment:
What if we do: `List.wrap(precolor)` and `List.wrap(postcolor)`. This way, when we call IO.ANSI.format_fragment, we can pass the whole list and it would also allow developers to set up multiple values. For example, `[:red, :bright]`.